lib/player.rb

```diff
@@ -0,0 +1,7 @@
+class Player
+  attr_reader :player
+
+  def initialize(player)
+    @player = player
```

Comment:
Change attribute, @player, to @name to avoid confusion

Reply:
That makes sense. Thanks